pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -91,9 +93,10 @@ effectiveStdenv.mkDerivation (finalAttrs: {
 
   buildInputs = optionals effectiveStdenv.isDarwin darwinBuildInputs
     ++ optionals cudaSupport cudaBuildInputs
-    ++ optionals mpiSupport mpi
+    ++ optionals mpiSupport [ mpi ]
     ++ optionals openclSupport [ clblast ]
     ++ optionals rocmSupport rocmBuildInputs
+    ++ optionals blasSupport [ openblas.dev ]
```

```suggestion
    ++ optionals blasSupport [ openblas ]
```
why is this `dev` need to link to the libraries too.

---

You are right, thanks. I updated this PR.